iceprog/Makefile

```diff
@@ -1,6 +1,20 @@
 include ../config.mk
-LDLIBS = -L/usr/local/lib -lm
-CFLAGS = -MD -O0 -ggdb -Wall -std=c99 -I/usr/local/include
+
+LIBFTDI_VERSION = $(shell $(PKG_CONFIG) --modversion libftdi1 2>/dev/null)
+ifneq ($(LIBFTDI_VERSION),)
+  LIBFTDI_NAME = ftdi1
+else
+  LIBFTDI_NAME = ftdi
+endif
+
```

Comment:
Wouldn't the following be better?

    LIBFTDI_NAME = $(shell $(PKG_CONFIG) --exists libftdi1 && echo ftdi1 || echo ftdi)

.. and maybe add this line after the `else` below, since it is only used when uname=Darwin.